src/graphql/FlowRun/table-task-runs.gql

```diff
@@ -9,7 +10,7 @@ query TableTaskRuns(
     id
     name
     task_runs(
-      where: { task: { name: { _ilike: $name } } }
+      where: { task: { name: { _ilike: $name } }, state: { _ilike: $state } }
```

Comment:
```suggestion
      where: { task: { name: { _ilike: $name } }, state: { _ilike: $state } }
```

I don't think we need to do a fuzzy search here, which I think will be slower; we can use `_eq` for state matching instead.